src/jag-vips-client/vipsords.yaml

```diff
@@ -99,6 +99,70 @@ paths:
         required: true
         type: string
         description: Implicit parameter
+  /vipsDisclosure/{documentId}/{authGuid}:
```

Comment:
Would you mind changing this to simply 'disclosure' instead of vipsDisclosure? Thanks, this is the path that I've already set up in the setDocument operation (not quite done). I assume the second operation to set the disclosed dtm will follow the same.

Reply:
updated